AST: InlineCommandComment

```
panic: unknown node type: 'InlineCommandComment 0x22e3510 <col:2, col:6> Name="NOTE" RenderNormal'

goroutine 34 [running]:
github.com/elliotchance/c2go/ast.Parse(0xc421798537, 0x46, 0x5fad87, 0x5)
	/home/konstantin/go/src/github.com/elliotchance/c2go/ast/ast.go:252 +0x273
main.convertLinesToNodes(0xc420547bd0, 0x49be, 0xdd3a, 0xc420473bd0, 0x47, 0xc420475f00)
	/home/konstantin/go/src/github.com/elliotchance/c2go/main.go:96 +0x196
main.convertLinesToNodesParallel.func1.2(0xc420412060, 0xc42000c008, 0xc420547bd0, 0x49be, 0xdd3a, 0x0)
	/home/konstantin/go/src/github.com/elliotchance/c2go/main.go:128 +0x59
created by main.convertLinesToNodesParallel.func1
	/home/konstantin/go/src/github.com/elliotchance/c2go/main.go:132 +0x1b6
```

--- a/ast/ast.go
+++ b/ast/ast.go
@@ -154,6 +154,8 @@ func Parse(fullline string) Node {
 		return parseIndirectFieldDecl(line)
 	case "InitListExpr":
 		return parseInitListExpr(line)
+	case "InlineCommandComment":
+		return parseInlineCommandComment(line)
 	case "IntegerLiteral":
 		return parseIntegerLiteral(line)
 	case "LabelStmt":

--- a/ast/block_command_comment.go
+++ b/ast/block_command_comment.go
@@ -1,5 +1,6 @@
 package ast
 
+// BlockCommandComment is a type of comment
 type BlockCommandComment struct {
 	Addr       Address
 	Pos        Position

--- a/ast/full_comment.go
+++ b/ast/full_comment.go
@@ -1,5 +1,6 @@
 package ast
 
+// FullComment is a type of comment
 type FullComment struct {
 	Addr       Address
 	Pos        Position

--- a/ast/html_end_tag_comment.go
+++ b/ast/html_end_tag_comment.go
@@ -1,5 +1,6 @@
 package ast
 
+// HTMLEndTagComment is a type of comment
 type HTMLEndTagComment struct {
 	Addr       Address
 	Pos        Position

--- a/ast/html_start_tag_comment.go
+++ b/ast/html_start_tag_comment.go
@@ -1,5 +1,6 @@
 package ast
 
+// HTMLStartTagComment is a type of comment
 type HTMLStartTagComment struct {
 	Addr       Address
 	Pos        Position

--- a/ast/inline_command_comment.go
+++ b/ast/inline_command_comment.go
@@ -1,0 +1,45 @@
+package ast
+
+type InlineCommandComment struct {
+	Addr       Address
+	Pos        Position
+	Other      string
+	ChildNodes []Node
+}
+
+func parseInlineCommandComment(line string) *InlineCommandComment {
+	groups := groupsFromRegex(
+		`<(?P<position>.*)> (?P<other>.*)`,
+		line,
+	)
+
+	return &InlineCommandComment{
+		Addr:       ParseAddress(groups["address"]),
+		Pos:        NewPositionFromString(groups["position"]),
+		Other:      groups["other"],
+		ChildNodes: []Node{},
+	}
+}
+
+// AddChild adds a new child node. Child nodes can then be accessed with the
+// Children attribute.
+func (n *InlineCommandComment) AddChild(node Node) {
+	n.ChildNodes = append(n.ChildNodes, node)
+}
+
+// Address returns the numeric address of the node. See the documentation for
+// the Address type for more information.
+func (n *InlineCommandComment) Address() Address {
+	return n.Addr
+}
+
+// Children returns the child nodes. If this node does not have any children or
+// this node does not support children it will always return an empty slice.
+func (n *InlineCommandComment) Children() []Node {
+	return n.ChildNodes
+}
+
+// Position returns the position in the original source code.
+func (n *InlineCommandComment) Position() Position {
+	return n.Pos
+}

--- a/ast/inline_command_comment.go
+++ b/ast/inline_command_comment.go
@@ -1,5 +1,6 @@
 package ast
 
+// InlineCommandComment is a type of comment
 type InlineCommandComment struct {
 	Addr       Address
 	Pos        Position

--- a/ast/inline_command_comment_test.go
+++ b/ast/inline_command_comment_test.go
@@ -1,0 +1,18 @@
+package ast
+
+import (
+	"testing"
+)
+
+func TestInlineCommandComment(t *testing.T) {
+	nodes := map[string]Node{
+		`0x22e3510 <col:2, col:6> Name="NOTE" RenderNormal`: &InlineCommandComment{
+			Addr:       0x22e3510,
+			Pos:        NewPositionFromString("col:2, col:6"),
+			Other:      "Name=\"NOTE\" RenderNormal",
+			ChildNodes: []Node{},
+		},
+	}
+
+	runNodeTests(t, nodes)
+}

--- a/ast/paragraph_comment.go
+++ b/ast/paragraph_comment.go
@@ -1,5 +1,6 @@
 package ast
 
+// ParagraphComment is a type of comment
 type ParagraphComment struct {
 	Addr       Address
 	Pos        Position

--- a/ast/param_command_comment.go
+++ b/ast/param_command_comment.go
@@ -1,5 +1,6 @@
 package ast
 
+// ParamCommandComment is a type of comment
 type ParamCommandComment struct {
 	Addr       Address
 	Pos        Position

--- a/ast/position.go
+++ b/ast/position.go
@@ -304,6 +304,8 @@ func setPosition(node Node, position Position) {
 		n.Pos = position
 	case *InitListExpr:
 		n.Pos = position
+	case *InlineCommandComment:
+		n.Pos = position
 	case *IntegerLiteral:
 		n.Pos = position
 	case *LabelStmt:

--- a/ast/text_comment.go
+++ b/ast/text_comment.go
@@ -1,5 +1,6 @@
 package ast
 
+// TextComment is a type of comment
 type TextComment struct {
 	Addr       Address
 	Pos        Position

--- a/ast/verbatim_line_comment.go
+++ b/ast/verbatim_line_comment.go
@@ -1,5 +1,6 @@
 package ast
 
+// VerbatimLineComment is a type of comment
 type VerbatimLineComment struct {
 	Addr       Address
 	Pos        Position

